### ISSUE TITLE
API Editor: Add UI support for Enum type

### ISSUE THREAD

**Issue by msavy** [Apicurio/apicurio-studio#140]
There doesn't seem to be any support for enum types. 

For example:

```
{
	"endpointContentType": {
		"title": "Endpoint Content Type",
		"description": "Gateway's content type when responding with errors/failures.",
		"enum": [
			"json",
			"xml"
		]
	}
}                         
```

![image](https://user-images.githubusercontent.com/423513/28131999-7e52362a-6733-11e7-9428-1dc869c0ef02.png)

You probably already know about this, but just adding a ticket to track it and/or remind you :-).

**Comment by EricWittmann** [comment]
Yeah lots of stuff not yet implemented in the designer.  We'll need to prioritize which things are most important.

Please definitely don't feel shy about creating tickets!  Much appreciated.

**Comment by 0815fox** [comment]
From my point of view this one should have a high priority ;-)
